src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -23,7 +23,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
                         ' It is not permissible to access the Left property of a BoundUserDefinedBinaryOperator unconditionally,
                         ' because that property can throw an exception if the operator expression is semantically invalid.
                         ' Fetching the left operand through IBinaryOperatorExpression is safe.
-                        If rightOperatorBinary.LeftOperand Is value.LeftOnTheRightOpt Then
+                        If rightOperatorBinary.LeftOperand Is Create(value.LeftOnTheRightOpt) Then
```

Comment:
rightOperatorBinary is IOperation. rightOperatorBinary.LeftOperand is IOperation.
value is BoundNode and value.LeftOnTheRightOpt is BoundNode. 

it used to work because BoundNode directly implemented IOperation. 
so "rightOperatorBinary.LeftOperand Is value.LeftOnTheRightOpt" would return true if both point to same instance of BoundNode which implements IOperation.

now, BoundNode and IOperation are different object so the above always return false.

the Create() method is IOperation factory entry point, IOperation Create(BoundNode).

so, if two were pointing to same BoundNode, Create will return same IOperation node which make "IS" work as it used to.